src/js/07-message-block.js

```diff
@@ -0,0 +1,11 @@
+;(function () {
+  // Ajust 'top' css property of sidebar TOC, when message block is present
+  var sidebar = document.querySelector('aside.toc.sidebar')
```

Comment:
**nitpick:** SonarCloud is complaining about the usage of `var` instead of `const` or `let`

![image](https://user-images.githubusercontent.com/27200110/221186651-ff090c07-a917-49c5-9dd5-fb1be4248592.png)

Reply:
I agree it would be better to do it at generation time. I'll try to do it...

Reply:
changed for `const`